### ISSUE TITLE
Retain valid fields in login and registration forms

### ISSUE THREAD

**Issue by cblanken** [cblanken/tesy-tagboard#87]
The login and registration forms both reset when an error happens. All the valid fields should be populated and only the invalid or incomplete fields should be reset.

<img width="805" height="703" alt="Image" src="https://github.com/user-attachments/assets/3b236b7d-9212-46c7-83fd-65d95df5dae6" />

**Comment by cblanken** [comment]
This will probably require overriding the the `SignupView` class from allauth. Specifically the `FormView.form_invalid` method.

See [this post](https://yourlabs.org/posts/2012-03-23-how-to-override-a-view-from-an-external-django-app/) for some details on how to override the necessary URL, and see the [Django docs](https://docs.djangoproject.com/en/6.0/topics/class-based-views/generic-editing/#form-handling-with-class-based-views) for more details.
